extensions/cnsr_ext.lua

```diff
@@ -115,20 +116,28 @@ function show_category_selection()
 	dlg:add_label("age restriction", 1, y, 1, 1)
 	dlg:add_button("set by parental guidence", click_restrict_age, x +3, y, 2, 1)
 	age_restriction_dropdown = create_drop_down(x-1, y, age_options)
-	
+
 	for idx, value in ipairs(CATEGORIES) do
 		dlg:add_label(value.description, 1, y+1, 1, 1)
 		dropdowns[idx] = create_drop_down(x, y+1, options)
 		y = y + 1
 	end
-  
 	dlg:add_label("Enter password:",1, y+1, 1, 1)
 	text_box = dlg:add_password("", 1, y + 2, 1, 1)
 	dlg:add_label("Hint: " .. pass_cfg["hint"],1, y + 3, 1, 1)
-	button_apply = dlg:add_button("Apply and save", click_play, x + 1, y + 3, 1, 1)
-	pass_status = dlg:add_label('',1, y + 4, 1, 1)
+
+	dlg:add_label("set offset:",1, y + 4,1,1)
+
+	key=Memory.get_key("offset")
```

Comment:
you can simple `key=Memory.get_key("offset") or 0`

Reply:
done